### ISSUE TITLE
bug(select): Non-native Selects do not announce state properly with screen readers

### ISSUE THREAD

**Issue by bmayen** [angular/components#30362]
### Is this a regression?

- [ ] Yes, this behavior used to work in the previous version

### Description

Reported by our accessibility team:
"When the selects are activated the state is not communicated. In JAWS neither state is communicated, in NVDA only the collapsed state is communicated."

### Reproduction

StackBlitz link: 
Steps to reproduce:
1. 
2.

### Expected Behavior

Should announce different states on state changes

### Actual Behavior

In JAWS neither state is communicated, in NVDA only the collapsed state is communicated

### Environment

- Angular: 18.x
- CDK/Material: 18.x
- Browser(s): Chromium
- Operating System (e.g. Windows, macOS, Ubuntu): Windows/OSX

**Comment by crisbeto** [comment]
We are closing this issue, because we've determined, using a semi-automated process, that it is no longer relevant, has low engagement, or it does not have enough information to proceed. Feel free to comment if you think the issue should be reopened, or open a new issue with the latest version of our code instead.
